### PR TITLE
Add empty env template

### PR DESCRIPTION
**Pull request by KPostOffice**

### CODE REVIEW

File: .env.template [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+THOTH_CEPH_SECRET_KEY=
+THOTH_CEPH_BUCKET=
+THOTH_CEPH_REGION=
+
+# Result Base
+THOTH_DEPLOYMENT_NAME=
+THOTH_CEPH_BUCKET_PREFIX=
+
+# Buildlogs
+thoth_deployment_name=
```
Comment (fridex) [inline .env.template]: `THOTH_DEPLOYMENT_NAME`?

**Comment by KPostOffice** [comment]
> LGTM.
> 
> `.env` to `.gitignore`?

It's already there

**Comment by fridex** [comment]
recheck

**Comment by KPostOffice** [comment]
recheck

**Pull request merged**